lib/kino_explorer/data_transform_cell.ex

```diff
@@ -837,14 +843,8 @@ defmodule KinoExplorer.DataTransformCell do
 
   defp update_data_options([operation], ctx, data_frame) do
     data_frames = ctx.assigns.data_frames
-    df = Enum.find_value(data_frames, &(&1.variable == data_frame && Map.get(&1, :data)))
-
-    data_options =
-      case df do
-        nil -> nil
-        %DataFrame{} -> DataFrame.dtypes(df) |> normalize_dtypes()
-        _ -> df |> DataFrame.new() |> DataFrame.dtypes() |> normalize_dtypes()
-      end
+    dtypes = Enum.find_value(data_frames, &(&1.variable == data_frame && Map.get(&1, :dtypes)))
```

Comment:
We always expect the key to be there, right?

```suggestion
    dtypes = Enum.find_value(data_frames, &(&1.variable == data_frame && &1.dtypes))
```